ci/unit/test_nlt_node.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This is a script to be run by the utils/node_local_test.sh to run a test
+# on a CI node.
+
+set -uex
+
+sudo bash -c 'echo 1 > /proc/sys/kernel/sysrq'
+sudo mkdir -p /mnt/daos
+
+#sudo mkdir -p "$DAOS_BASE"
+
+sudo ln -sF `readlink -f build` "$DAOS_BASE"
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Use $(..) instead of legacy `..`. [SC2006]